jlm/llvm/ir/attribute.hpp

```diff
@@ -117,7 +117,8 @@ class attribute
     VScaleRange,
     LastIntAttr,
 
-    EndAttrKinds ///< Sentinel value useful for loops
+    EndAttrKinds, ///< Sentinel value useful for loops
+    NoFPClass
```

Comment:
I doubt it that this comes after the sentinel value for loops.

Reply:
Moved

Reply:
This was not moved, it is still after the sentinel.

Reply:
Sorry, should be fixed now